charts/k8ssandra/templates/stargate/stargate.yaml

```diff
@@ -48,6 +48,10 @@ spec:
         - name: {{ .Release.Name }}-{{ include "k8ssandra.datacenterName" . }}-stargate
           image: {{ $image }}
           imagePullPolicy: {{ .Values.stargate.imagePullPolicy }}
+          lifecycle:
+            preStop:
+              exec:
+                command: [ "sleep", "20" ]
```

Comment:
It might be good to make `20` the default value for an entry in values.yaml, perhaps something like `stargate.preStop.delay`.